module/enrichers.mjs

```diff
@@ -0,0 +1,195 @@
+ * </a>
+ * ```
+ */
+export async function enrichSave(config, label) {
+  const ability = config.shift();
+  const dc = Number.isNumeric(config[0]) ? parseInt(config.pop()) : null;
+
+  const abilityConfig = CONFIG.DND5E.abilities[ability];
+  if ( !abilityConfig ) return console.log(`Ability ${ability} not found`);
+
+  if ( !label ) {
+    const ability = abilityConfig.label;
+    label = game.i18n.format("EDITOR.DND5E.Inline.Save", { ability });
+    if ( dc ) label = game.i18n.format("EDITOR.DND5E.Inline.DC", { dc, check: label });
+  }
+
+  return createRollLink(label, { type: "save", ability, dc });
+}
+
+/* -------------------------------------------- */
+
+/**
+ * Enrich a skill check link. Unlike the `@Check` enricher, this will use the player's default ability
+ * or allow for selecting any associated ability to perform the skill check.
+ * @param {string[]} config     Configuration data.
+ * @param {string} label        Optional label to replace default text.
+ * @returns {HTMLElement|null}  A HTML link if the save could be built, otherwise null.
+ *
+ * TODO: Add some examples
+ */
+export async function enrichSkill(config, label) {
+  const skill = config.shift();
+  const dc = Number.isNumeric(config[0]) ? parseInt(config.pop()) : null;
+
+  const skillConfig = CONFIG.DND5E.skills[skill];
+  if ( !skillConfig ) return console.log(`Skills ${skill} not found`);
+
+  if ( !label ) {
+    label = game.i18n.format("EDITOR.DND5E.Inline.Check", { ability: skillConfig.label });
+    if ( dc ) label = game.i18n.format("EDITOR.DND5E.Inline.DC", { dc, check: label });
+  }
+
+  return createRollLink(label, { type: "skill", skill, dc });
+}
+
+/* -------------------------------------------- */
+
+/**
+ * Create a rollable link.
+ * @param {string} label  Label to display.
+ * @param {object} dataset
+ * @param {string} dataset.type     Type of rolling action to perform (e.g. `check`, `save`, `skill`).
+ * @param {string} dataset.ability  Ability key to roll (e.g. `dex`, `str`).
+ * @param {string} dataset.skill    Skill key to roll (e.g. `ath`, `med`).
+ * @param {number} dataset.dc       DC of the roll.
+ * @returns {HTMLElement}
+ */
+export function createRollLink(label, dataset) {
+  const link = document.createElement("a");
+  link.classList.add("roll-link");
+  for ( const [key, value] of Object.entries(dataset) ) {
+    if ( value ) link.dataset[key] = value;
+  }
+  link.innerHTML = `<i class="fa-solid fa-dice-d20"></i> ${label}`;
+  return link;
+}
+
+/* -------------------------------------------- */
+/*  Actions                                     */
+/* -------------------------------------------- */
+
+/**
+ * Perform the provided roll action.
+ * @param {Event} event  The click event triggering the action.
+ * @returns {Promise|void}
+ */
+export function rollAction(event) {
+  let target = event.target;
+  if ( !target.classList.contains("roll-link") ) target = target.closest(".roll-link");
```

Comment:
I think this is unnecessary because `Element#closest` can return itself if it matches the provided selector.
```suggestion
  const target = event.target.closest(".roll-link");
```